src/adapters/github.js

```diff
@@ -58,7 +58,7 @@ class GitHub extends PjaxAdapter {
 
   // @override
   getCreateTokenUrl() {
-    return `${location.protocol}//${location.host}/settings/tokens/new`
+    return `${location.protocol}//${location.host}/settings/tokens/new?scopes=public_repo&description=OctoTree browser extension`
```

Comment:
Good idea. I think many people only start realizing they need token when they hit a private repo and Octotree prompts for the token. Therefore, maybe a good idea to automatically select all repos to begin with. Also, please change description to "Octotree browser extension".

Reply:
Good points, done!